### ISSUE TITLE
discrepancy between paper and benchmarks

### ISSUE THREAD

**Issue by proteneer** [deepchem/deepchem#431]
In paper it was mentioned that for tox21 the test set has an AUC of ~0.84 but in the benchmarks it's listed as 0.798 (random splitting) - what changed?

**Comment by rbharath** [comment]
@proteneer Thanks for the report here!

@miaecle Would you mind checking out what the source of the discrepancy is?

**Comment by miaecle** [comment]
@proteneer Thanks for pointing out the issue!
In the benchmark table we only included results for train set and valid set, 0.798(actually this is the performance under index splitting, 0.827 for random splitting) should correspond to the valid set performance in the paper.
Also, results in the paper come from a full run of benchmark, while table on the github page only includes numbers we get when the dataset is merged into the dataset. Performances of graphconv are still a bit unstable, there maybe a ±1% differences between runs.

**Comment by lilleswing** [comment]
Other benchmarks not hitting expectations.

| Dataset | Model | Split | Metric | Retrieved Value | Desired Value | 
| -- | -- | -- | -- | -- | -- |
|muv|Multitask network|Index splitting|test_score|0.749443916693|0.764|
|muv|logistic regression|Index splitting|test_score|0.756238252709|0.766|
|muv|graph convolution|Index splitting|test_score|0.791876998348|0.823|
|pcba|Multitask network|Index splitting|train_score|0.814176263345|0.826|
|pcba|Multitask network|Index splitting|test_score|0.788107828923|0.802|
|clintox|Multitask network|Index splitting|test_score|0.819248826291|0.83|
|clintox|robust MT-NN|Index splitting|test_score|0.81514084507|0.827|
|nci|graphconv regression|Index splitting|train_score|0.112239244388|0.123|
|nci|graphconv regression|Index splitting|test_score|0.0464834859959|0.048|
|qm7b|NN regression|Index splitting|train_score|0.876963119457|0.931|
|qm7b|NN regression|Index splitting|test_score|0.790498748877|0.803|
|qm7b|NN regression|Index splitting|train_score|0.878199577192|0.931|
|qm7b|NN regression|Index splitting|test_score|0.791941826351|0.803|
|qm7b|NN regression|Index splitting|train_score|0.879837090375|0.931|
|qm7b|NN regression|Index splitting|test_score|0.792008993439|0.803|
|qm7|NN regression|Index splitting|test_score|0.97433488412|0.986|
|sampl|NN regression|Index splitting|test_score|0.755635996|0.764|
|sampl|NN regression|Random splitting|test_score|0.761702572858|0.83|
|sampl|NN regression|Scaffold splitting|train_score|0.880153572737|0.891|
|nci|graphconv regression|Index splitting|train_score|0.117258262877|0.123|
|nci|graphconv regression|Index splitting|train_score|0.115851838028|0.123|
|nci|graphconv regression|Index splitting|test_score|0.0463009131706|0.048|
|qm7|NN regression|Index splitting|test_score|0.958073353377|0.986|
|nci|graphconv regression|Random splitting|train_score|0.106167884328|0.117|
|nci|graphconv regression|Random splitting|test_score|0.0706557598031|0.076|

**Comment by miaecle** [comment]
@lilleswing Thanks for getting this, looks like Index splitting is causing most of the discrepancies, will look into it! qm7 & qm7b are using different hyperparameters #415, which should be the cause.

**Comment by rbharath** [comment]
@lilleswing and I chatted offline. I think the 1% is relative difference currently. Moving to > 1% absolute difference should remove many (but not all) of the failing datasets.

**Comment by rbharath** [comment]
Closing this issue since I think we've resolved these discrepancies.
